dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/registry/impl/ServiceLocatorServer.scala

```diff
@@ -31,7 +31,7 @@ class ServiceLocatorServer extends Closeable {
     gatewayImpl:            String
   ): Unit =
     synchronized {
-      require(server == null, "Service locator is already running on " + server.mainAddress)
+      require(server == null, "Service locator is already running on " + server.asInstanceOf[Server].mainAddress)
```

Comment:
`server` is already a `Server`, in this diff.  Is this no longer required?

Reply:
Good call!